Podfile

```diff
@@ -1,13 +1,18 @@
-# Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
+use_frameworks!
 
-target 'Example' do
-  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
-  use_frameworks!
+def common
+    pod 'Texture', '~> 2.5'
+    pod 'RxSwift', '~> 4.0'
+    pod 'RxCocoa', '~> 4.0'
+    pod 'Differentiator', '~> 3.0'
+end
 
-  pod 'Texture'
-  pod 'RxSwift'
-  pod 'RxCocoa'
-  pod 'Differentiator'
+target 'Example' do
+    common
+end
 
+target 'RxASDataSources' do
```

Comment:
I think I used CocoaPod for Example only. For RxAsDataSources, I'm using Carthage with `--use-submodules ` for manage dependencies?